### PR TITLE
[OSDOCS#11637] Compact clusters on OpenStack

### PR DESCRIPTION
**Pull request by maxwelldb**
Version(s): 4.17

Issue: https://issues.redhat.com/browse/OSDOCS-11637

Link to docs preview: https://80595--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_openstack/installing-openstack-three-node.html

QE review:
- [x] QE has approved this change.

Additional information:

- An example of existing AWS three-node cluster docs: https://80595--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_aws/installing-aws-three-node

### CODE REVIEW

**Comment by itzikb-redhat** [comment]
Looks ok from my side

**Review by mandre — COMMENTED** [review]
LGTM.

Is there a table that summarize all features that each platform supports? Should we mark the three-node cluster configuration supported on OpenStack there too? I kind of remember one at some point, I don't know if it still exists.

**Comment by maxwelldb** [comment]
> LGTM.
> 
> Is there a table that summarize all features that each platform supports? Should we mark the three-node cluster configuration supported on OpenStack there too? I kind of remember one at some point, I don't know if it still exists.

Are you thinking of this?: https://docs.openshift.com/container-platform/4.16/installing/overview/installing-preparing.html#supported-installation-methods-for-different-platforms

If so, seems like three-node clusters aren't part of them for any platform, yet.

**Comment by mandre** [comment]
> > LGTM.
> > Is there a table that summarize all features that each platform supports? Should we mark the three-node cluster configuration supported on OpenStack there too? I kind of remember one at some point, I don't know if it still exists.
> 
> @mandre Are you thinking of this?: https://docs.openshift.com/container-platform/4.16/installing/overview/installing-preparing.html#supported-installation-methods-for-different-platforms
> 
> If so, seems like three-node clusters aren't part of them for any platform, yet.

My memory is fuzzy and that could very well be it indeed.

File: installing/installing_openstack/installing-openstack-three-node.adoc [hunk 1]
```diff
@@ -0,0 +1,17 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="installing-openstack-three-node"]
```
Comment (dfitzmau) [inline installing/installing_openstack/installing-openstack-three-node.adoc:2]: I see that this module is a almost identical to the [GCP module](https://80595--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_gcp/installing-gcp-three-node); maybe they could be merged and ifdef/ifndef statements could be used to consolidate two modules into one. This is not mandatory requirement.

File: installing/installing_openstack/installing-openstack-three-node.adoc [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+= Installing a three-node cluster on OpenStack
+include::_attributes/common-attributes.adoc[]
+:context: installing-openstack-three-node
+
+toc::[]
+
+In {product-title} version {product-version}, you can install a three-node cluster on {rh-openstack-first}. A three-node cluster consists of three control plane machines, which also act as compute machines. This type of cluster provides a smaller, more resource efficient cluster, for cluster administrators and developers to use for testing, development, and production.
+
+You can install a three-node cluster using either installer-provisioned or user-provisioned infrastructure.
```
Comment (dfitzmau) [inline installing/installing_openstack/installing-openstack-three-node.adoc:11]:
```suggestion
You can install a three-node cluster by using either installer-provisioned or user-provisioned infrastructure.
```

**Review by dfitzmau — APPROVED** [review]
/remove-label peer-review-in-progress

/label peer-review-done

**Comment by maxwelldb** [comment]
@itzikb-redhat @mandre Are there any docs tweaks you anticipate requesting before GA? If not, I can merge this. If maybe so, I'll leave it open through docs freeze.

**Comment by mandre** [comment]
No tweaks needed that I can think of.

**Comment by itzikb-redhat** [comment]
me too

**Pull request merged**